### ISSUE TITLE
Add Slinkity

### ISSUE THREAD

**Issue by seancdavis** [seancdavis/seancdavis-com#360]
A few months back I explored what it would take to incorporate a component-driven approach for this project (#162). I was leaning toward Astro, but decided not to pursue.

Since then, Slinkity has mad a splash in the 11ty community and is almost exactly what I need. I also think it would keep me involved with the community. So I'd like to get it incorporated.

I put together a little PoC and there's a bit of work to make it happen. While I'd like to replace components as needed and have some that are React and some that aren't, it's not going to be super easy to have my own JavaScript and let Slinkity do the building. So it may be better to just _rip the band-aid_ off and overhaul the whole thing.

**Comment by seancdavis** [comment]
I put my quick PoC in a branch — `360/slinkity`.

**Comment by seancdavis** [comment]
I built up quite a bit on `360/slinkity` and got a good feel for Slinkity. It's cool, but it's not _really_ going to solve the overall issue of waiting for pages to be built.

I've spent a lot of time thinking about my next move over the last six months. I was hoping Slinkity could get me there, but closing it out isn't worth the effort.

I will likely end up evaluating Next.js, Gatsby, Remix, or perhaps SvelteKit as the next iteration of this site. But I'm not ready to do that just yet. For now I need to show that I can get the increased usage with these new content types. Then I can think about what an overhaul would look like.

The change will be painful no matter what, and definitely less so if I did it now. But I also don't think it's worth waiting to see if this content even works. It's a likely scenario that some of the content doesn't work and is a drag to create. I'd rather learn that now and make the right decision when I have more users.
